pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -597,62 +615,62 @@ within modules. For example, @racket[define] within @racket[begin-for-syntax] cr
 @racket[make-base-namespace] and @racket[eval] provide access to
 top-level variables in higher @tech{phases}, while module
 @tech{instantiations} (triggered by @racket[require]) relative to such
-top-levels are in corresponding higher @tech{phase}s.
+top-levels are in correspondingly higher @tech{phase}s.
 
 @;- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 @subsection[#:tag "separate-compilation"]{The Separate Compilation Guarantee}
 
 When a module is compiled, its @tech{phase} 1 is instantiated. This
 can, in turn, trigger the transitive instantiation of many other
 modules at other phases, including phase 1. Racket provides a very
-strong guarantee about this instantiation called "The Separate
-Compilation Guarantee":
+strong guarantee about this instantiation called ``The Separate
```

Comment:
maybe add `@index["separate compilation guarantee"]{The Separate ....}`

Reply:
Will do.